recipes/papillon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "papillon" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "7505f977ed053b9eff70ba3a6a0787b227f36c419d8d742e3c74d09ebc7bc9c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
You can't have `noarch: python` if you're using the `skip` to exclude versions of python. Drop the `noarch` line.